Sources/App/Core/AppEnvironment.swift

```diff
@@ -75,7 +77,17 @@ extension AppEnvironment {
                 .get("ROLLBAR_LOG_LEVEL")
                 .flatMap(AppError.Level.init(rawValue:)) ?? .critical },
         shell: .live,
-        siteURL: { Environment.get("SITE_URL") ?? "http://localhost:8080" }
+        siteURL: { Environment.get("SITE_URL") ?? "http://localhost:8080" },
+        twitterCredentials: {
+            guard let apiKey = Environment.get("TWITTER_API_KEY"),
+                  let apiKeySecret = Environment.get("TWITTER_API_SECRET"),
+                  let accessToken = Environment.get("TWITTER_ACCESS_TOKEN_KEY"),
+                  let accessTokenSecret = Environment.get("TWITTER_ACCESS_TOKEN_SECRET")
```

Comment:
I've added the Blahblaher account's keys and secrets to `DEV` - `PROD` has no keys yet.